src/main/c/ssl.c

```diff
@@ -1191,6 +1191,7 @@ TCN_IMPLEMENT_CALL(jlong /* SSL * */, SSL, newSSL)(TCN_STDARGS,
                                                    jlong ctx /* tcn_ssl_ctxt_t * */,
                                                    jboolean server) {
     tcn_ssl_ctxt_t *c = J2P(ctx, tcn_ssl_ctxt_t *);
+    int * handshakeCount = malloc(sizeof(int));
```

Comment:
Is `uint32_t` (or `uint16_t`) a better choice for a count?  Are we intending to use negative numbers?